app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Attributes
+  attribute :text, :text
+end
```

Comment:
- [ ] You have probably forgotten to add the increment method to this, kind do and note the recommendation I pointed out in the post file.